game/missiongenerator/aircraft/waypoints/seadingress.py

```diff
@@ -1,7 +1,8 @@
 import logging
 
 from dcs.point import MovingPoint
-from dcs.task import AttackGroup, OptECMUsing, WeaponType
+from dcs.task import AttackGroup, EngageGroup, OptECMUsing, WeaponType
+from game.data.weapons import WeaponType as DcsWeaponType
```

Comment:
The imported name here is the opposite of our usual convention. `Dcs*` is used elsewhere for the pydcs type. Either name this `LiberationWeaponType` or name the other one `DcsWeaponType` so we don't get confused looking at the usage sites.

Reply:
oh, you're completly right! swapping it :)